### PR TITLE
Email validation

### LINKED ISSUE #1: Email validation needed
We need the components/modules that can actually take an email & validate it's an email address

### PR DESCRIPTION
**Pull request by ay-a-git**
fixes #1

### CODE REVIEW

File: nimbusCBT/client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
```
Comment (crc8109) [inline nimbusCBT/client/.gitignore:10]: Is this supposed to be here?

Comment (crc8109) [reply nimbusCBT/client/.gitignore:10]: I know the folder exists, but looks like we might have to remove this line?

File: nimbusCBT/client/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+import EmailValidator from "./components/EmailValidation.jsx";
+
+function App() {
+  const [count, setCount] = useState(0);
+
+  return (
+    <div className="App">
+      <h1>Email thing</h1>
```
Comment (Lito-Frito) [inline nimbusCBT/client/src/App.jsx]: Is this PR going to become the module we later use in the app? I.e. are we just going to reuse this code when the app is more fleshed out and we need to incorporate this feature? 

If so, then maybe we should make some of the boilerplate more official looking/parameterized so it'll be easier to implement in the future. We can discuss Sunday depending your answer

File: nimbusCBT/client/src/components/EmailValidation.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { useFormik } from "formik";
+
+const validate = (values) => {
+  const errors = {};
+
+  if (!values.email) {
+    errors.email = "Required";
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
```
Comment (Lito-Frito) [inline nimbusCBT/client/src/components/EmailValidation.jsx]: :man_cook: :pinched_fingers:

**Review by Lito-Frito — COMMENTED** [review]
See comments. I think next step after addressing those would be to actually create the script that GH or whatever would run to install all this. I actually just thought of a good reason as to why. We can talk more Sunday

**Review by Lito-Frito — DISMISSED** [review]
LGTM. 

There is a lot of files that seem auto generated. Is There any way to reduce the noise on the repo by having some on gitignore and having them generated when a user runs yarn or something? E.g. something like [this](https://github.com/crc8109/COVID19-World-Tracker).

If not, feel free to approve the merge.

**Review by ay-a-git — COMMENTED** [review]
99% node module packages. Is this too much of a rabbit hole to focus on right now?

**Review by Lito-Frito — APPROVED** [review]

**Pull request merged**